README.md

```
@@ -13,88 +13,87 @@ auto-completion, go-to-definition, and more all from within Neovim 💻🔧
+  - [License](#license)
 
 ## Dependencies
 
-You must install the following plug-ins:
-
-- [`seblyng/roslyn.nvim`](https://github.com/seblyng/roslyn.nvim) for general
-  functionality
-- `html-lsp` for completions and formatting. You can install and configure it
-  via `mason` and `nvim-lspconfig`.
+- [`roslyn`](https://github.com/crashdummyy/roslynlanguageserver) - The C# language server. Required for all C# and Razor/Blazor integrations.<br/>
+    [-> Jump to installation instructions](#roslyn)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)